### DISCUSSION TITLE
OT 1.3 core tasks Tracker

### DISCUSSION THREAD

**Discussion by testter2** [openthread/openthread#7760] [Q&A] [answered]
Hi,

Is there any opened issue that have all core tasks related to Openthread 1.3 as it's done with Openthread 1.2 in the following link
[Thread 1.2.0 Core Implementation](https://github.com/openthread/openthread/issues/4327)

this will be very helpful in tracking all main tasks related to OT1.3 for anyone interested in it.

Thanks

**Comment by jwhui** [comment, marked as answer]
Yes, see the [Thread Spec Implementation Project page](https://github.com/orgs/openthread/projects/1).

**Comment by testter2** [comment]
Thank you for quick answer @jwhui , this will be very helpful.

**Comment by abtink** [comment]
You can also check (which conatins same info)
- https://github.com/openthread/openthread/issues/7460  
 
(though we may move to use the project (only) in future)

**Comment by testter2** [comment]
Thanks @abtink .

I can see on openthrad.io that OpenThread implements all features defined in the Thread 1.1.1 Specification but not 1.2 so this should be changed to 1.2 now, right ?

another question : Will be any coming stable release soon in the following link ?
[Thread References](https://github.com/openthread/openthread/releases)

Thanks.

**Reply by jwhui** [reply]
OpenThread is currently certified for Thread 1.1. When OpenThread has been certified under 1.2 or any later version, we will update the listed certification version.

**Comment by testter2** [comment]
Thanks for all explanations.
